project_status/data/data.xml

```diff
@@ -0,0 +1,17 @@
+<odoo>

```

Comment:
@patrickrwilson  `<odoo noupdate="1">`

Reply:
Can you explain more of what's needed here? I'm not sure what you are wanting changed.

Reply:
I agree: otherwise if the user customizes the default data, the changes are reversed on the next module upgrade.

Reply:
He is using
 `<data noupdate="1">` 
which is valid in this case

Reply:
@patrickrwilson @dreispt  In the latest version, You should directly write `<odoo noupdate="1">`, No need `<data>` tag.
So, remove  `<data>` tag and using `<odoo noupdate="1">`